MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/IReflPresenter.h

```diff
@@ -35,14 +31,17 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 File change history is stored at: <https://github.com/mantidproject/mantid>.
 Code Documentation is available at: <http://doxygen.mantidproject.org>
 */
-class IReflPresenter {
+class IReflPresenter : public IReflOuterPresenter {
```

Comment:
No, I guess we don't really need to. I've modified this bit so that now it is the concrete presenter, `ReflMainViewPresenter` who inherits from `WorkspaceReceiver`